src/bin/bindgen.rs

```diff
@@ -304,14 +70,15 @@ pub fn main() {
         }
     }
 
-    let (options, out) = parse_args_or_exit(bind_args);
-
-    let mut bindings = Bindings::generate(options, None)
-        .expect("Unable to generate bindings");
-
-    bindings.write_dummy_uses()
-        .expect("Unable to write dummy uses to file.");
-
-    bindings.write(out)
-        .expect("Unable to write bindings to file.");
+    match builder_from_flags(env::args()) {
+        Ok((builder, output)) => {
+            let mut bindings = builder.generate()
+                .expect("Unable to generate bindings");
+            bindings.write(output)
+                .expect("Unable to write output");
+            bindings.write_dummy_uses()
+                .expect("Unable to write dummy uses to file.");
+        }
+        Err(error) => println!("{}", error)
```

Comment:
nit: We most likely want to exit with a nonzero error code here. Panicking may be ok, other kind of handling even better :)